src/Aspire.Dashboard/README.md

```diff
@@ -0,0 +1,11 @@
+# .NET Aspire Dashboard
+
+## As a dotnet tool
+
+The dashboard can be packaged and executed as a `dotnet tool`.
```

Comment:
Apologies, the docs weren't updated when the tool support was removed.

We removed it because we no longer needed it for our scenario and (from memory) it conflicted with other packaging requirements.

Do you have a use case for a tool? If so, could you file an issue explaining how you'd use it?

Reply:
I think you should be able to get that executable working and not need a dotnet tool. If not, please open a new issue.